ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Types.hs

```diff
@@ -937,9 +944,9 @@ data TracePeerSelection peeraddr =
      -- Peer Sharing
      --
 
-     -- | target known peers, actual known peers, peers available for
-     -- peer sharing, peers selected for peer sharing
-     | TracePeerShareRequests     Int Int (Set peeraddr) (Set peeraddr)
+     -- | target known peers, actual known peers, number of peers to request,
+     -- peers available for peer sharing, peers selected for peer sharing
+     | TracePeerShareRequests     Int Int PeerSharingAmount (Set peeraddr) (Set peeraddr)
```

Comment:
I think changing the tracer type is a breaking change that should be mentioned in the changelog.